moengage-segment-kotlin-destination/build.gradle.kts

```diff
@@ -10,10 +10,10 @@ val libVersionName = project.findProperty("VERSION_NAME") as String
 
 android {
     namespace = "com.segment.analytics.kotlin.destinations.moengage"
-    compileSdk = 34
+    compileSdk = 35
```

Comment:
use the configuration from internal catalog.

Reply:
done